src/routes/(main)/auth/register/+page.svelte

```diff
@@ -0,0 +1,106 @@
+<script lang="ts">
+	import Form from '$lib/component/Form';
+	import type { PageData } from './$types';
+	import { schema } from './form-schema';
+	import Icon from '@iconify/svelte';
+	import { superForm } from 'sveltekit-superforms';
+	import { arktypeClient } from 'sveltekit-superforms/adapters';
+
+	let { data }: { data: PageData } = $props();
+
+	const superform = superForm(data.form, { validators: arktypeClient(schema) });
+</script>
+
+<svelte:head>
+	<title>Rejestracja - HoshiAnime</title>
+	<meta
+		name="description"
+		content="Załóż konto w HoshiAnime, aby korzystać z pełni możliwości naszej platformy."
+	/>
+</svelte:head>
+
+<div
+	class="card m-auto max-h-4/5 w-[calc(100%-2rem)] max-w-3xl self-center bg-base-300 shadow-sm drop-shadow-xl sm:card-side"
+>
+	<figure class="w-full not-sm:aspect-video sm:max-w-2/6">
+		<picture>
```

Comment:
Using `<picture>` with different aspect-ratio images for mobile and desktop to improve the visuals.